src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableDepthOfClasses.java

```diff
@@ -36,21 +36,21 @@ void method() {
 
     class r {
         class k {
-            int p = 12;
+            int s = 12;
         }
     }
 
-    static class p {
+    static class s {
```

Comment:
This is the only "necessary" change, but I renamed all `p` identifiers in order to ensure that coverage of any identifier-based analysis is not impacted.